src/app.jsx

```diff
@@ -459,6 +464,162 @@ class Application extends React.Component {
+                    key: container_key,
+                    Id: key,
+                    IsService: false,
+                    IsInfra: false,
+                    Name: quadlet.name,
+                    ImageName: quadlet.image,
+                    NetworkSettings: {
+                        Ports: [],
+                    },
+                    Mounts: [],
+                    Config: {
+                        Labels: {
+                            PODMAN_SYSTEMD_UNIT: key,
+                        },
+                    },
+                    // HACK, hardcoded, needs mapping from systemd and O(n) calls
+                    State: {
+                        Status: 'exited'
+                    }
+                };
+
+                const found_pod = podNameServiceMap[quadlet.pod];
+                if (found_pod) {
+                    container.Pod = found_pod;
+                }
+                copyQuadletContainers[container.key] = container;
+            }
+
+            const users = prevState.users.map(u => u.uid === con.uid ? { ...u, quadletsLoaded: true } : u);
+            return { quadletContainers: copyQuadletContainers, quadletPods: copyQuadletPods, users };
+        });
+    }
+
+    async subscribeDaemonReload(con) {
+        // We don't support subscribing on reload events for "other" users.
+        if (con.uid !== 0 && con.uid !== null) {
+            return;
+        }
+
+        debug('subscribe daemon reload', con);
+
+        const options = con.uid === 0 ? { bus: "system", superuser: "try" } : { bus: "session" };
+        const subscribe = (client) => {
+            const subscription = client.subscribe({ interface: "org.freedesktop.systemd1.Manager", member: "Reloading" }, (_path, _iface, _signal, [reloading]) => {
+                if (!reloading)
+                    this.initQuadlets(con);
+            });
+
+            this.setState(prevState => {
+                const users = prevState.users.map(u => u.uid === con.uid ? { ...u, dbus: { client, subscription } } : u);
+                return { users };
+            });
+        };
+
+        let client = null;
+        const user = this.state.users.find(u => u.uid === con.uid);
+
+        // don't add multiple Reload subscriptions
+        if (user?.dbus?.subscription) {
+            return;
+        }
+
+        if (user?.dbus) {
+            client = user.dbus.client;
+        } else {
+            client = cockpit.dbus("org.freedesktop.systemd1", options);
+        }
+
+        client.call("/org/freedesktop/systemd1", "org.freedesktop.systemd1.Manager", "Subscribe", []).then(() => {
+            subscribe(client);
+        })
+                .catch(err => {
+                    if (err.name === "org.freedesktop.systemd1.AlreadySubscribed") {
+                        subscribe(client);
+                    } else {
+                        client.close();
+                        console.error(`Cannot subscribe systemd reload event for ${con.uid}`);
```

Comment:
These 2 added lines are not executed by any test.  [Details](https://logs-cockpit.apps.ocp.cloud.ci.centos.org/pull-2266-e4572c69-20251008-083258-fedora-42-devel/Coverage/lcov/github-pr.diff.gcov.html)